test/functional/editor/defaults_spec.lua

```diff
@@ -95,6 +95,60 @@ describe('default', function()
     end)
   end)
 
-  -- describe('key mappings', function()
-  -- end)
+  describe('key mappings', function()
+    describe('unimpaired-style mappings', function()
+      it('do not show a full stack trace #30625', function()
+        n.clear({ args_rm = { '--cmd', 'comclear | mapclear | mapclear!' } })
```

Comment:
```suggestion
        n.clear({ args_rm = { '--cmd' } })
```

Reply:
Why? Won't that remove all `--cmd` arguments? I only want to remove the `:mapclear` ones.